modules/prompt/functions/prompt_sorin_setup

```diff
@@ -123,6 +129,10 @@ function prompt_sorin_setup {
   zstyle ':prezto:module:git:info:keys' format \
     'status' '%b %p %c:%s%A%B%S%a%d%m%r%U%u'
 
+
+  # Set python-info format
+  zstyle ':prezto:module:python:info:virtualenv' format '%f%F{208}(%v)%F{7} '
```

Comment:
We have to be careful with the colors that we use (particularly for default theme). Best to stick to the 16 terminal colors for maximum compatibility. We might have to have ellipsis for very long string (again, as safety for default theme).